analiza_danych_python/ubezpieczenia_app/funkcje_kor_bledow.py

```diff
@@ -0,0 +1,99 @@
+import pandas as pd
+import os
+import numpy as np
+
+
+def choose_state():
+    with open("dane/states.csv", "r") as plik:
+        states = [linia.rstrip() for linia in plik]
+    print("\nLista state codes:")
+    print(states)
+    state_code = input("\nPodaj kod stanu, w którym chcesz się ubezpieczyć:")
+    if state_code in states:
+        return state_code
+    else:
+        print("\nNiepoprawny state code. Wybierz z listy.")
+        state_code = input("Podaj kod stanu, w którym chcesz się ubezpieczyć:")
+        return state_code
+
+
+def choose_age():
+    while True:
+        age_input = input("\nPodaj swój wiek:")
+        if age_input.isnumeric() and int(age_input)<120:
+            if int(age_input) <= 20:
+                age = '0-20'
+            elif int(age_input) >= 65:
+                age = '65 and over'
+            else:
+                age = age_input
+            return age
+            break
+        else:
+            print("co poszlo nie tak . Podaj wiek w latach w zakresie lat 0-120 .")
+
+
+
+def choose_option():
+    opt_dict = {
+        1:"IndividualRate",
+        2:"Couple",
+        3:"PrimarySubscriberAndOneDependent",
+        4:"PrimarySubscriberAndTwoDependents",
+        5:"PrimarySubscriberAndThreeOrMoreDependents",
+        6:"CoupleAndOneDependent",
+        7:"CoupleAndTwoDependents",
+        8:"CoupleAndThreeOrMoreDependents"
+        }
+    print("\nPoniższa lista prezentuje dostępne opcje ubezpieczenia rodzinnego:")
+    print("\n 2 - Couple\n 3 - Primary Subscriber and One Dependent\n 4 - Primary Subscriber and Two Dependents\n 5 - Primary Subscriber and Three or More Dependents\n 6 - Couple And One Dependent\n 7 - Couple And Two Dependents\n 8 - Couple And Three or More Dependents\n")
+    option = int(input("Z powyższej listy wybierz numer opcji ubezpieczenia:"))
+    if option in range(2,9):
+        return option
+    else:
+        print("\nNie ma takiej opcji. Wybierz numer z listy")
+        option = int(input("Z powyższej listy wybierz numer opcji ubezpieczenia:"))
+        return option
+
+def get_data():
+    family = input("Czy chcesz się ubezpieczyć z rodziną? y/n:")
+    if family == 'y':
+        age = 'Family Option'
+        option = choose_option()
+    else:
```

Comment:
Tu trzeba dodać sprawdzenie błędów. Jak ktoś wpisze "tak" to wtedy wejdzie do else